mappings/net/minecraft/entity/EntityType.mapping

```diff
@@ -151,7 +151,7 @@ CLASS aid net/minecraft/entity/EntityType
 		ARG 1 world
 	METHOD d getEntityClass ()Laik;
 	METHOD e getTranslationKey ()Ljava/lang/String;
-	METHOD f getTextComponent ()Ljm;
+	METHOD f getComponent ()Ljm;
```

Comment:
Imo, getters like this should have more specific names. Something like `getName` could work

Reply:
I agree. `getComponent` is very confusing, especially if you start thinking about entity component systems.